include/shared.h

```diff
@@ -344,6 +347,23 @@ static inline void ft_stop(void)
 	clock_gettime(CLOCK_MONOTONIC, &end);
 	opts.options &= ~FT_OPT_ACTIVE;
 }
+
+/* Set the FI_MSG_PREFIX mode bit in the given fi_info structure and also set
+ * the option bit in the given opts structure. If using ft_getinfo, it will
+ * return -ENODATA if the provider clears the application requested mdoe bit.
+ */
+static inline void ft_force_prefix(struct fi_info *info, struct ft_opts *opts)
+{
+	info->mode |= FI_MSG_PREFIX;
+	opts->force_prefix = 1;
+}
+
+static inline bool ft_check_prefix(struct fi_info *info)
+{
+	return (info->tx_attr->mode & FI_MSG_PREFIX) &&
```

Comment:
nit: given that ft_force_prefix() sets force_prefix, it makes sense to have ft_check_prefix() check the force_prefix option for symmetry.  Other than that, the changes are fine.  Thanks